packages/codebytes/src/index.tsx

```diff
@@ -8,6 +8,16 @@ import React, { useState } from 'react';
 import { languageOption } from './consts';
 import { Editor } from './editor';
 
+export interface CodeByteEditorProps {
+  text: string;
+  language: languageOption;
+  hideCopyButton: boolean;
+  onCopy?: (text: string, language: string) => void;
+  isIFrame?: boolean;
+  snippetsBaseUrl?: string /* TODO in DISC-353: Determine best way to host and include snippets endpoint for both staging and production in both the monolith and next.js repo. */;
```

Comment:
Surfacing from Zoom: you mentioned prop drilling in general, and there's going to be a good amount of that going on... might be a good idea to use a React Context for `language`, `snippetsBaseUrl`, `text`, and anything else that would need to be drilled a bunch?

Reply:
@JoshuaKGoldberg, going to add an argument here for not using context :) 
- `language` and `text` are only used to initialize local internal states, and the states are passed down further to the `Editor`, so it's not _exactly_ drilling props down. 
- `snippetsBaseUrl` is being passed down two levels from the parent, to CodebyteEditor, to the Editor component. 

If there were 3-4 levels of prop drilling, I would agree, but we might not be there in this case.

Reply:
cc @zippyzow